actions/FirefallClient.js

```diff
@@ -42,29 +54,32 @@ class FirefallClient {
           n: 1,
         },
       })
-      .json();
+      .json()
+      .catch((error) => {
+        // console.error('Error generating completion:', error);
```

Comment:
I suggest using a logger here, as this information is required to troubleshoot action errors.

Reply:
For this I considered two approaches:

1. Passing the logger object from the "AuthAction" and reusing it in the FirefallClient.
2. Creating a new logger object in the FirefallClient by passing the LOG_LEVEL to the FirefallClient.

I think #1 makes more sense, what do you suggest?

Thanks.

Reply:
The logging has already been implemented. Just merge this one with the main branch: https://github.com/adobe/aem-genai-assistant/pull/195